rpmlb/downloader/base.py

```diff
@@ -38,6 +38,11 @@ def get_instance(cls, name):
         return instance
 
     def run(self, work, **kwargs):
+        is_resume = 'resume' in kwargs and kwargs['resume']
```

Comment:
There is a better python idiom for that:
```python
is_resume = kwargs.get('resume', False)
```

The `dict.get(key, default)` returns the value with `key` if such key is in the dictionary, else it returns the `default` value. [docs](https://docs.python.org/3/library/stdtypes.html?highlight=dict.get#dict.get)